File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = {}
        for i in nums:
            count[i] = count.get(i,0)+1
        count = sorted(count.items(),key=lambda item:item[1],reverse=True)
        # print(count)
        res = []
        for i in range(0,k):
            res.append(count[i][0])
        return res

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_map<int,int> count;
        for(int i = 0; i < nums.size(); i++){
            count[nums[i]]++;
            if(count[nums[i]]>1){
                return true;
            }
        }
        return false;
    }
};